libs/deposits/src/lib/use-submit-deposit.tsx

```diff
@@ -72,19 +71,17 @@ export const useSubmitDeposit = () => {
       });
 
       if (matchingDeposit && matchingDeposit.event.__typename === 'Deposit') {
```

Comment:
nitpick (non-blocking): any way we can get rid of this magic string

Reply:
Not without a ts-ignore. We only get Deposit events via the subscription but because the event field is a union type of all possible bus event types this is needed to appease TS